src/main/java/lotto/domain/LottoTickets.java

```diff
@@ -0,0 +1,58 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class LottoTickets {
+    private static final int BONUS_CONDITION = 5;
+    private static final String DELIMITER = "\n";
+    private List<LottoTicket> lottoTickets;
+
+    LottoTickets(List<LottoTicket> lottoTickets) {
+        this.lottoTickets = Collections.unmodifiableList(lottoTickets);
+    }
+
+    public static LottoTickets createLottoTickets(Money money) {
+        List<LottoTicket> lottoTickets = new ArrayList<>();
+        for (int i = 0; i < money.ticketQuantity(); i++) {
```

Comment:
```suggestion
        for (int i = 0, end = money.ticketQuantity(); i < end; i++) {
```

와 같이 변경한다면 불필요한 메서드 호출을 제거할 수 있습니다.